### PR TITLE
Fix uyuni proxy sync

### PR DESCRIPTION
**Pull request by maximenoel8**
## What does this PR change?

Uyuni product synchronized for proxy is not correct.

## GUI diff

No difference.

- [x] **DONE**

## Documentation
- No documentation needed: only internal and user invisible changes

- [x] **DONE**

## Test coverage
ℹ️ If a major new functionality is added, it is **strongly recommended** that tests for the new functionality are added to the Cucumber test suite
- No tests: already covered

- [x] **DONE**

## Links

Port:
Issue: 

- [x] **DONE**

## Changelogs

Make sure the changelogs entries you are adding are compliant with https://github.com/uyuni-project/uyuni/wiki/Contributing#changelogs and https://github.com/uyuni-project/uyuni/wiki/Contributing#uyuni-projectuyuni-repository

If you don't need a changelog check, please mark this checkbox:

- [x] No changelog needed

If you uncheck the checkbox after the PR is created, you will need to re-run `changelog_test` (see below)

## Re-run a test

If you need to re-run a test, please mark the related checkbox, it will be unchecked automatically once it has re-run:

- [ ] Re-run test "changelog_test"
- [ ] Re-run test "backend_unittests_pgsql"
- [ ] Re-run test "java_lint_checkstyle"
- [ ] Re-run test "java_pgsql_tests"
- [ ] Re-run test "ruby_rubocop"
- [ ] Re-run test "schema_migration_test_pgsql"
- [ ] Re-run test "susemanager_unittests"
- [ ] Re-run test "javascript_lint"
- [ ] Re-run test "spacecmd_unittests"

# Before you merge

Check [How to branch and merge properly](https://github.com/uyuni-project/uyuni/wiki/How-to-branch-and-merge-properly)!

### CODE REVIEW

File: testsuite/features/build_validation/reposync/srv_sync_all_products.feature [hunk 1]
```diff
@@ -794,7 +794,7 @@ Feature: Synchronize products in the products page of the Setup Wizard
 @uyuni
 @proxy
   Scenario: Add Uyuni Leap 15.5 Proxy, including Uyuni Client Tools
-    When I use spacewalk-common-channel to add all "leap15.5" channels with arch "x86_64"
+    When I use spacewalk-common-channel to add all "uyuni-proxy" channels with arch "x86_64"
```
Comment (szachovy) [inline testsuite/features/build_validation/reposync/srv_sync_all_products.feature:797]: I don't get it here, why in `Uyuni Leap 15.5 Proxy` we are adding micro5.5 channels instead?

File: testsuite/features/support/constants.rb [hunk 2]
```diff
@@ -1338,14 +1338,10 @@
...
-        opensuse_leap15_5-x86_64
-        opensuse_leap15_5-uyuni-client-x86_64
-        opensuse_leap15_5-x86_64-backports-updates
-        opensuse_leap15_5-x86_64-non-oss
-        opensuse_leap15_5-x86_64-non-oss-updates
-        opensuse_leap15_5-x86_64-sle-updates
-        opensuse_leap15_5-x86_64-updates
-        uyuni-proxy-devel-leap-x86_64
+        opensuse_micro5_5-x86_64
+        opensuse_micro5_5-x86_64-sle-updates
```
Comment (srbarrios) [inline testsuite/features/support/constants.rb:1342]: Naming seems correct https://github.com/uyuni-project/uyuni/blob/master/utils/spacewalk-common-channels.ini#L1411-L1449 👍🏼

**Review by srbarrios — APPROVED** [review]

File: testsuite/features/build_validation/reposync/srv_sync_all_products.feature (hunk 1 above)
Comment (maximenoel8) [reply testsuite/features/build_validation/reposync/srv_sync_all_products.feature:797]: Good catch, the scenario name is not correct.

[the remark this answers is not in the document]
Comment (maximenoel8) [reply testsuite/features/build_validation/reposync/srv_sync_all_products.feature:797]: This is done during combustion.

[the remark this answers is not in the document]
Comment (maximenoel8) [reply testsuite/features/build_validation/reposync/srv_sync_all_products.feature:797]: Do you mean the repostores from additional_repository variable ?
We do know add those reposiroties into combustion: https://github.com/uyuni-project/sumaform/blob/addeb4382c1c77c2e84f8e7644d66319fa28361e/backend_modules/libvirt/host/combustion#L97C1-L105C5

That what trigger me to open this issue: https://github.com/SUSE/spacewalk/issues/25609

**Pull request merged**